src/config.h

```diff
@@ -55,6 +55,7 @@ extern "C" {
 #define CLOUD_STATUS_OFFLINE    "offline"
 #define CLOUD_DISCONNECT_REASON "disconnection-reason"
 #define BOOT_RETRY_WAIT         "boot-time-retry-wait"
+#define TOKEN_SERVER_URL        "https://issuer.xmidt.comcast.net:8080/issue"
```

Comment:
This might have to be an input command line argument

Reply:
incorporated.